src/gen-objects.ts

```diff
@@ -443,6 +447,7 @@ export function addImageDefinition(target: PresSlide, opt: ImageProps) {
 		flipH: opt.flipH || false,
 		transparency: opt.transparency || 0,
 		objectName: objectName,
+		shadow: opt.shadow ?? { type: 'none' }
```

Comment:
I think you can potentially inline `correctShadowOptions` here.
```suggestion
		shadow: correctShadowOptions(opt.shadow)
```

Reply:
From what I understand, the method mutates the `opt.shadow` object so I'm coalescing a possible `undefined` to a default shadow (which is type: none)

Reply:
`correctShadowOptions` returns `undefined` when invalid options are passed; in `gen-xml.ts` there's a check if `slideItemObj.options.shadow` is present or not.